android/libraries/rib-base/src/test/java/com/badoo/ribs/core/routing/backstack/BackStackManagerTest.kt

```diff
@@ -50,6 +43,10 @@ class BackStackManagerTest {
         }
         backStackRibConnector = mock()
         setupBackStackManager(timeCapsuleEmpty)
+
+        backStackManager.state.backStack.forEach {
+            it.builtNodes = listOf()
```

Comment:
Why not just call reset?

Reply:
Otherwise there's nothing in the builtNodes field